invenio.cfg

```diff
@@ -550,3 +550,9 @@ OPENAIRE_DIRECT_INDEXING_ENABLED = True
 """Enable sending published records for direct indexing at OpenAIRE."""
 
 RDM_RECORDS_SERVICE_COMPONENTS = DefaultRecordsComponents + [OpenAIREComponent]
+
+MAIL_DEFAULT_SENDER = "Zenodo <info@zenodo.org>"
```

Comment:
@lnielsen what's the `noreply@cern.ch` address equivalent we should be using for Zenodo?

Reply:
I'm checking with Emmanuel.

Reply:
We can use ``noreply@zenodo.org``

Reply:
Modified the config for Zenodo-RDM 👍